frontend/src/components/Form/Dropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+import React, { FC } from "react";
+
+interface iDropDown {
+  users: [];
```

Comment:
It's an array of what? Type also the values of the array.
E.g.: 
- `['tom', 'john']` = `string[]`
- `[{id: 1, name: 'tom'}, {id: 2, name: 'john'}]` = `{id: number, name: string}[]`

You could even put the object inside of a `Type` like so:
```
Type UserType = {id: number, name: string}

interface iDropdown {
 users: UserType[]
}

```

Reply:
Oh I actually see that you already have the types for the array value `iUser`. 
so 
```
interface iDropdown {
 users: iUser[]
}
```